### DISCUSSION TITLE
About displaying the historical breadcrumb

### DISCUSSION THREAD

**Discussion by is01** [combostrap/combo#87] [Q&A]
I would like to place the historical breadcrumb at the top of the main page for quick access. 
It would be nice if it could be displayed in parallel with the hiearchical breadcrumb, is that possible?

I think that the topic of this discussion is somewhat related with  https://github.com/ComboStrap/combo/discussions/20 .

**Comment by gerardnico** [comment]
Hi is01,

This is for now unfortunately not possible.

Historical breadcrumb is an admin component, already available in the browser, change for every page (ie no publishing cache, bad for performance)

Combostrap is a publishing platform and therefore will never provide it as component for this reason.

Quick navigation is the key here. I agree that we can do better and miss a quick navigation component keyboard based. 

The browser that you uses may have it.

Best
Nico

**Reply by is01** [reply]
Thanks for the reply.
I don't understand the exact concept of a publishing platform, but I understand Combostrap's policy.
I think it would be useful if there were other quick navigation options.

Thanks!

**Reply by gerardnico** [reply]
Publishing means that the document is published (in the context of a wiki, don't change frequently, can be cached, created once, and distributed everywhere)

For combostrap, with a dynamic component, the [public cache component](https://combostrap.com/cache/what-is-the-public-page-cache-vn1g4dpl) would not work as the page should be rendered for each request.

Dokuwiki is a little bit strange in that optic because it mixes an application (ie creating the doc, backend) and the publishing (ie rendering, front end)

You don't see it but we have clients that cache the pages in multiple locations through proxies.

**Reply by is01** [reply]
Thanks for elaborating. 
I use a private wiki for my notes, and I update it very frequently. That would be a bit different to intended use.

Thank you for all your efforts on this nice plugin and its performance improvements.

**Comment by gerardnico** [comment]
I did not use them but when I talk about browser plugins for history.

Here is an example with Popup: [Recent history](https://chromewebstore.google.com/detail/recent-history/fbmkfdfomhhlonpbnpiibloacemdhjjm)
